python/dgl/distributed/dist_graph.py

```diff
@@ -381,6 +377,10 @@ def __init__(self, server_id, ip_config, num_servers,
                 self.init_data(name=str(data_name), policy_str=data_name.policy_str,
                                data_tensor=node_feats[name])
                 self.orig_data.add(str(data_name))
+            del node_feats
+            gc.collect()
```

Comment:
Look like L370-381 is to get the feature names but it loads the entire feature data instead. Is it possible to read them from some metadata files?

Reply:
no. feature data is really used in `self.init_data()`. we really need to load the feature data.